src/azure-cli/azure/cli/command_modules/keyvault/commands.py

```diff
@@ -61,33 +62,39 @@ def load_command_table(self, _):
 
     # Management Plane Commands
     with self.command_group('keyvault', kv_vaults_sdk, client_factory=keyvault_client_vaults_factory) as g:
-        g.custom_command('create', 'create_keyvault',
+        g.custom_command('create', 'create_keyvault', supports_no_wait=True,
                          doc_string_source='azure.mgmt.keyvault.v' + mgmt_api_version + '.models#VaultProperties')
-        g.custom_command('recover', 'recover_keyvault')
+        g.custom_command('recover', 'recover_keyvault', supports_no_wait=True)
         g.custom_command('list', 'list_keyvault')
         g.show_command('show', 'get')
+
+        g.command('purge', 'begin_purge_deleted', supports_no_wait=True)
```

Comment:
Are these new commands ? It seems that we didn't add test for them.

Reply:
@arrownj No, it's an old command. Actually `purge` command has been already covered: https://github.com/Azure/azure-cli/blob/f3db6bd039592bac8c65400f303c5a09ead7c877/src/azure-cli/azure/cli/command_modules/keyvault/tests/latest/test_keyvault_commands.py#L221.

Reply:
seems line71~73 is duplicate with line81~83

Reply:
> seems line 71 ~ 73 is duplicated with line 81 ~ 83

@arrownj Good catch! Removed.